pinot-core/src/main/java/com/linkedin/pinot/core/segment/SimpleSegmentNameGenerator.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (C) 2014-2016 LinkedIn Corp. (pinot-core@linkedin.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.linkedin.pinot.core.segment;
+
+import com.linkedin.pinot.common.utils.SegmentNameBuilder;
+import com.linkedin.pinot.core.segment.creator.AbstractColumnStatisticsCollector;
+
+
+public class SimpleSegmentNameGenerator implements SegmentNameGenerator {
+  public String getSegmentName(AbstractColumnStatisticsCollector statsCollector, SegmentNameConfig config) throws Exception {
+    String timeColumn = config.getTimeColumnName();
+    String segmentName;
+    if (timeColumn != null && timeColumn.length() > 0) {
+      final Object minTimeValue = statsCollector.getMinValue();
+      final Object maxTimeValue = statsCollector.getMaxValue();
+      segmentName = SegmentNameBuilder
```

Comment:
Ideally we should not have SegmentNamBuilder class at all, but I think it is being used in the third-eye code, so we cannot remove it now.